### PR TITLE
add divider on protocol market

### PR DESCRIPTION
**Pull request by Brakmel**
After
Before

### CODE REVIEW

File: components/organisms/ProtocolMarket.tsx [hunk 1]
```diff
@@ -19,15 +20,20 @@ export const ProtocolTokensMarket = () => {
...
-                                        <View flexDirection={"row"} alignItems={"center"} marginTop={5}>
+                                        <View flexDirection={"row"} alignItems={"center"} marginTop={5}> 
                                             <TokenIcon address={aaveAddress} size={13}/>
                                             <SecondaryText style={{ marginLeft: 5 }}>
                                                 Reward Aave
                                             </SecondaryText>
                                         </View>
+                                        <View style={styles.divider}>
+                                                <SecondaryText>
+
```
Comment (LuisB0605) [inline components/organisms/ProtocolMarket.tsx]: Can we change this?

File: components/organisms/ProtocolMarket.tsx [hunk 2]
```diff
@@ -19,15 +20,20 @@ export const ProtocolTokensMarket = () => {
...
+                                        <View flexDirection={"row"} alignItems={"center"} marginTop={5}> 
                                             <TokenIcon address={aaveAddress} size={13}/>
                                             <SecondaryText style={{ marginLeft: 5 }}>
                                                 Reward Aave
                                             </SecondaryText>
                                         </View>
+                                        <View style={styles.divider}>
+                                                <SecondaryText>
+                                                    
+                                                </SecondaryText>
```
Comment (samhv) [inline components/organisms/ProtocolMarket.tsx]: Can we remove `SecondaryText` here?

File: components/organisms/ProtocolMarket.tsx (hunk 1 above)
Comment (LuisB0605) [reply components/organisms/ProtocolMarket.tsx]: There is a blank space

**Review by LuisB0605 — CHANGES_REQUESTED** [review]

File: components/organisms/ProtocolMarket.tsx [hunk 3]
```diff
@@ -43,3 +49,10 @@ export const ProtocolTokensMarket = () => {
         <TokenRowMarket address={"0x..."} name={"USD Coin"} volumen={"5.57"} borrowAPY={"3.26"} depositAPY={"2.19"} rewardAaveAPY={"0.96"} />
     </View>;
 };
+const styles = StyleSheet.create({
+    divider: {
+        height: 0.5,
```
Comment (samhv) [inline components/organisms/ProtocolMarket.tsx]: Can we change it to 1? In some cases using 0.5 it doesn't show anything on some devices

File: components/organisms/ProtocolMarket.tsx [hunk 4]
```diff
@@ -43,3 +49,10 @@ export const ProtocolTokensMarket = () => {
...
+        width: 275,
```
Comment (samhv) [inline components/organisms/ProtocolMarket.tsx]: I don't think we have to use a fixed value here. Can we just remove it? If we can't, then can we just use `100%` here?

File: components/organisms/ProtocolMarket.tsx [hunk 5]
```diff
@@ -43,3 +49,10 @@ export const ProtocolTokensMarket = () => {
...
+        backgroundColor:"grey"
```
Comment (samhv) [inline components/organisms/ProtocolMarket.tsx]: Let's use something from `STYLES`, or let's create something here

**Review by samhv — CHANGES_REQUESTED** [review]
A couple comments, let's address them! 🤗

File: components/organisms/ProtocolMarket.tsx (hunk 2 above)
Comment (Brakmel) [reply components/organisms/ProtocolMarket.tsx]: Ok

File: components/organisms/ProtocolMarket.tsx (hunk 1 above)
Comment (Brakmel) [reply components/organisms/ProtocolMarket.tsx]: Ok

File: components/organisms/ProtocolMarket.tsx (hunk 3 above)
Comment (Brakmel) [reply components/organisms/ProtocolMarket.tsx]: Ready

**Review by samhv — APPROVED** [review]
nice, lgtm!

**Review by samhv — CHANGES_REQUESTED** [review]
Actually, can we add some margin on top of the bar pls? we need like 3 or 5 pixels pls

**Review by samhv — APPROVED** [review]
lgtm!

**Review by LuisB0605 — APPROVED** [review]

**Review by samhv — CHANGES_REQUESTED** [review]
The line isn't touching the edges, I think it's because the screen has padding. The movement here is remove the padding from the container and add it to the children

**Pull request closed without merging**